crates/ruma-events/src/message.rs

```diff
@@ -197,6 +203,7 @@ impl From<TextContentBlock> for MessageEventContent {
 /// To construct a `TextContentBlock` with custom MIME types, construct a `Vec<TextRepresentation>`
 /// first and use its `::from()` / `.into()` implementation.
 #[derive(Clone, Debug, Default, Serialize, Deserialize)]
+#[serde(transparent)]
```

Comment:
This change seems unrelated to the stabilization. Are there more changes like that in this PR?

Reply:
In my opinion this is not totally unrelated. This type is being stabilized so we better fix any issues with it.

For example the addition of `TopicContentBlock` below is not really about stabilization, the serialization of the unstable type was wrong.

I can make these changes in separate commits if you prefer.

Reply:
Yeah, it would make review easier to have the semantic changes separated from the stabilization.

Reply:
Done.